firebase-functions/ktx/src/androidTest/kotlin/com/google/firebase/functions/ktx/CallTests.kt

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.functions.ktx
+
+import com.google.common.truth.Truth.assertThat
+import com.google.firebase.FirebaseApp
+import com.google.firebase.ktx.Firebase
+import com.google.firebase.ktx.app
+import com.google.android.gms.tasks.Tasks
+import org.junit.BeforeClass
+import org.junit.Test
+import org.junit.runner.RunWith
+import android.support.test.InstrumentationRegistry
+import android.support.test.runner.AndroidJUnit4
+
+@RunWith(AndroidJUnit4::class)
+class CallTests {
+    companion object {
+        lateinit var app: FirebaseApp
+
+        @BeforeClass @JvmStatic fun setup() {
+            FirebaseApp.initializeApp(InstrumentationRegistry.getContext())
+            app = FirebaseApp.getInstance()
+        }
+    }
+
+    @Test
+    fun testDataCall() {
+        val functions = Firebase.functions(app)
+        val input = hashMapOf(
+            "bool" to true,
+            "int" to 2,
+            "long" to 3L,
+            "string" to "four",
+            "array" to listOf(5, 6),
+            "null" to null
+        )
+
+        var function = functions.getHttpsCallable("dataTest")
+        val result = function(input)
+        val actual = Tasks.await(result).getData()
+
+        assertThat(actual is Map<*, *>).isTrue()
```

Comment:
Can be simplified as `assertThat(actual).isInstanceOf(Map::class.java)` ?

Reply:
You're right, much better. Thanks!